### ISSUE TITLE
The plugin crashes if you export './index'

### ISSUE THREAD

**Issue by w0rp** [bcherny/tslint-no-circular-imports#17]
I was just using the plugin and I had an index file which contained the following line by mistake.

```ts
export * from './index'
```

This code crashes the plugin with the following error:

```
TypeError: Cannot read property 'getStart' of undefined
```

I have created a project which demonstrates the bug here: https://github.com/w0rp/no-circular-import-index-error

You can run `yarn && node_modules/.bin/tslint -p .` on that project to see the error, or you can create a similar file and try it yourself. I think that bug should be fixed, and it should be reported as an error.
